tilequeue/rawr.py

```diff
@@ -249,6 +251,11 @@ def __call__(self):
                 self.log_exception(e, 'queue read')
                 continue
 
+            if not msg_handle:
+                # this gets triggered when no messages are returned
+                # TODO do we want to sleep here for a small amount of time?
```

Comment:
I've since configured the receive wait time on the queue itself to be the maximum of 20 seconds, so at least it won't busy loop. Depending on how fast we get through diffs typically, it might make sense to sleep here for a longer time period though. Otherwise we'll just be making more sqs read calls than otherwise necessary.

Reply:
It might be more obvious and less prone to accidental breakage if we pass in the receive wait param to the `receive_message()` call ([documented here](http://boto3.readthedocs.io/en/latest/reference/services/sqs.html#SQS.Client.receive_message)).

Reply:
Yea, I thought it was actually better to push that configuration to the queue though, because if we wanted to change our minds, we just have to update the queue configuration rather than redeploy.

That being said, I don't think we'll ever change our minds in this case, and having it in the code will at least guarantee this behavior. I agree that it will make it more obvious as to what's going on as well.

Updated in ceb9f79.